regen/opcode.pl

```diff
@@ -78,84 +78,84 @@ BEGIN
 
 # Format is "this function" => "does these op names"
 my @raw_alias = (
-		 Perl_do_kv => [qw( keys values )],
-		 Perl_unimplemented_op => [qw(padany custom)],
-		 # All the ops with a body of { return NORMAL; }
-		 Perl_pp_null => [qw(scalar regcmaybe lineseq scope)],
-
-		 Perl_pp_goto => ['dump'],
-		 Perl_pp_require => ['dofile'],
-		 Perl_pp_untie => ['dbmclose'],
-		 Perl_pp_sysread => {read => '', recv => '#ifdef HAS_SOCKET'},
-		 Perl_pp_sysseek => ['seek'],
-		 Perl_pp_ioctl => ['fcntl'],
-		 Perl_pp_ssockopt => {gsockopt => '#ifdef HAS_SOCKET'},
-		 Perl_pp_getpeername => {getsockname => '#ifdef HAS_SOCKET'},
-		 Perl_pp_stat => ['lstat'],
-		 Perl_pp_ftrowned => [qw(fteowned ftzero ftsock ftchr ftblk
-					 ftfile ftdir ftpipe ftsuid ftsgid
- 					 ftsvtx)],
-		 Perl_pp_fttext => ['ftbinary'],
-		 Perl_pp_gmtime => ['localtime'],
-		 Perl_pp_semget => [qw(shmget msgget)],
-		 Perl_pp_semctl => [qw(shmctl msgctl)],
-		 Perl_pp_ghostent => [qw(ghbyname ghbyaddr)],
-		 Perl_pp_gnetent => [qw(gnbyname gnbyaddr)],
-		 Perl_pp_gprotoent => [qw(gpbyname gpbynumber)],
-		 Perl_pp_gservent => [qw(gsbyname gsbyport)],
-		 Perl_pp_gpwent => [qw(gpwnam gpwuid)],
-		 Perl_pp_ggrent => [qw(ggrnam ggrgid)],
-		 Perl_pp_ftis => [qw(ftsize ftmtime ftatime ftctime)],
-		 Perl_pp_chown => [qw(unlink chmod utime kill)],
-		 Perl_pp_link => ['symlink'],
-		 Perl_pp_ftrread => [qw(ftrwrite ftrexec fteread ftewrite
- 					fteexec)],
-		 Perl_pp_shmwrite => [qw(shmread msgsnd msgrcv semop)],
-		 Perl_pp_syswrite => {send => '#ifdef HAS_SOCKET'},
-		 Perl_pp_defined => [qw(dor dorassign)],
+                 Perl_do_kv => [qw( keys values )],
+                 Perl_unimplemented_op => [qw(padany custom)],
+                 # All the ops with a body of { return NORMAL; }
+                 Perl_pp_null => [qw(scalar regcmaybe lineseq scope)],
+
+                 Perl_pp_goto => ['dump'],
+                 Perl_pp_require => ['dofile'],
+                 Perl_pp_untie => ['dbmclose'],
+                 Perl_pp_sysread => {read => '', recv => '#ifdef HAS_SOCKET'},
+                 Perl_pp_sysseek => ['seek'],
+                 Perl_pp_ioctl => ['fcntl'],
+                 Perl_pp_ssockopt => {gsockopt => '#ifdef HAS_SOCKET'},
+                 Perl_pp_getpeername => {getsockname => '#ifdef HAS_SOCKET'},
+                 Perl_pp_stat => ['lstat'],
+                 Perl_pp_ftrowned => [qw(fteowned ftzero ftsock ftchr ftblk
+                                         ftfile ftdir ftpipe ftsuid ftsgid
+                                         ftsvtx)],
```

Comment:
lol

Reply:
There's no code change, just whitespace. Be sure to tick "Hide whitespace changes".

Reply:
Sorry, laughing that the previous indentation of this line was a (1) space character followed by tabs.